ui/component/walletSendTip/view.jsx

```diff
@@ -97,12 +110,14 @@ function WalletSendTip(props: Props) {
             <React.Fragment>
               {claimIsMine || isSupport
                 ? __(
-                    'This will increase the overall bid amount for %title%, which will boost its ability to be discovered while active.',
-                    { title }
```

Comment:
We need to keep the current structure of the string here so it works well with translations.

You should be able to keep everything the same, but also add the channel name after the title

```
...
{ title || `@${channelName}` }
```

Reply:
```
`This will increase the overall bid amount for ${title ? '%title%' : `%channelName%`}, which will boost its ability to be discovered while active.`,
{ title, channelName }
```

Is it the correct approach? 
I've tested it and it shows the localization string on the modal that has a title but not when using channelName.

```
...
{ title || `@${channelName}` }
```
Or using that is the correct one?

Reply:
The second one is correct